src/applications/edu-benefits/1995/submitForm.js

```diff
@@ -17,6 +16,9 @@ const submitForm = (form, formConfig) => {
     activeDuty: form.data.isActiveDuty ? 'Yes' : 'No',
     calledActiveDuty: form.data.isActiveDuty ? 'Yes' : 'No',
     preferredContactMethod: form.data.preferredContactMethod,
+    'edu-exhaustedAllBenefits': form.data['view:exhaustionOfBenefits']
```

Comment:
I think this is how you would access the data, see https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/1995/pages/stemEligibility.js#L18
```suggestion
    'edu-exhaustedAllBenefits': form.data['view:exhaustionOfBenefits']['view:exhaustionOfBenefits']
```

Reply:
I'm just accessing the `view:exhaustionOfBenefits` property in the form's data. I don't think `view:exhaustionOfBenefits` is an object that contains it's own `view:exhaustionOfBenefits`.

I think the reason they are doing it like that in https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/1995/helpers.jsx#L20
is because they need to access `view:determineEligibility` within `form[view:determineEligibility]`. I don't know why it's written like that.